WolvenKit.RED4.CR2W/Types/cp77/DeviceMappinsContainer.cs

```diff
@@ -38,7 +38,7 @@ public CBool UseNewFocusMappin
 
 		[Ordinal(3)] 
 		[RED("offsetValue")] 
-		public CFloat OffsetValue
+		public CFloat OffsetValue_
```

Comment:
What happened here?

Reply:
I wrote the wrong Regex here https://github.com/WopsS/RED4.RTTIDumper/commit/9652b9b0d8349d763b81138e1ed22bfff050dd97#diff-fe9c8992bfbddd91835ccb3908e64f54e6cd3b8e321a5d619ef9412e4830142fL558 `\SetValue\b` instead of `\bSetValue\b`